### ISSUE TITLE
Sending Throughput in the Metrics

### ISSUE THREAD

**Issue by SumantaSarathiChakraborty** [johrstrom/jmeter-prometheus-plugin#70]
Is there already a way to generate the Throughput metrics?

**Comment by johrstrom** [comment]
Yea when you measure ResponseSize. In histograms and quantiles there's a _sum metric. Use the rate function over that _sum metric.

**Comment by GiovanniPaoloGibilisco** [comment]
Maybe having some default metrics will help some people to get started.

**Comment by johrstrom** [comment]
Yep! I'd think so. I'm going to close this though as I believe it's been answered.
